docs/app/js/highlight-angular.js

```diff
@@ -53,6 +53,7 @@ angular.module('docsApp')
 
           var highlightedCode = hljs.highlight(attr.language || attr.lang, lines.join('\n'), true);
           highlightedCode.value = highlightedCode.value
+            // eslint-disable-next-line no-div-regex
             .replace(/=<span class="hljs-value">""<\/span>/gi, '')
```

Comment:
should be possible to do `\=` here

Reply:
That gives me `no-useless-escape` error. I think that I am just going to disable `no-div-regex` globally.